homeassistant/components/home_connect/coordinator.py

```diff
@@ -208,7 +208,7 @@ async def _event_listener(self) -> None:
                             events = self.data[event_message_ha_id].events
                             for event in event_message.data.items:
                                 event_key = event.key
-                                if event_key in SettingKey:
+                                if event_key in SettingKey.__members__:
```

Comment:
We need to compare against the members not the member names.
```suggestion
                                if event_key in SettingKey.__members__.values():
```